SchedulePlatform.Api/Controllers/MenuController.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using SchedulePlatform.Api.Mappings;
+using SchedulePlatform.Api.Models;
+using SchedulePlatform.Models.Entities;
+using SchedulePlatform.Service.Interfaces;
+
+namespace SchedulePlatform.Api.Controllers
+{
+	[Route("Api/[Controller]")]
+
+	[ApiController]
+
+	public class MenuController:ControllerBase
+	{
+		private readonly IMenuService _service;
+
+		public MenuController(IMenuService service)
+		{
+			_service = service;
+		}
+
+		[HttpGet]
+
```

Comment:
This line here is useless and makes the code harder to read. Please remove it. (This comment applies to many places so please remove it from everywhere).